components/FormAddNurse/FormAddNurse.js

```diff
@@ -0,0 +1,130 @@
+import useSWR from "swr";
+import { useState } from "react";
+import { useRouter } from "next/router";
+import {
+  FormContainer,
+  InputGroup,
+  Label,
+  Input,
+  Select,
+  CancelButtonStyled,
+  AddButtonStyled,
+  HeaderStyled,
+} from "./FormAddNurse.styled";
+
+export default function FormAddNurse() {
+  const { mutate } = useSWR("/api/nurses"); //check if db has new data - re-validation!
+  const router = useRouter();
+
+  const [selectedImage, setSelectedImage] = useState(getRandomImageURL());
+
+  function getRandomImageURL() {
+    const randomNumber = Math.floor(Math.random() * 70) + 1; //Random number for the nurse photo from 1 to 70
+    return `https://i.pravatar.cc/150?img=${randomNumber}`;
+  }
```

Comment:
This function can be outside the component, since it doesn't need anything from its scope:

```js
function getRandomImageURL() { ... }

export default function FormAddNurse() { ... }
```